### PR TITLE
Add links to the Mac tutorial in the main tutorial

### PR DESCRIPTION
**Pull request by mheon**
As the title says. Should reduce confusion for people trying on OS X for the first time.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: docs/tutorials/README.md [hunk 1]
```diff
@@ -11,3 +11,7 @@ Learn how to setup Podman and perform some basic commands with the utility.
 **[Basic Setup and Use of Podman in a Rootless environment.](https://github.com/containers/libpod/blob/master/docs/tutorials/rootless_tutorial.md).
```
Comment (TomSweeneyRedHat) [inline docs/tutorials/README.md]: I know it's old, but could you add two asterisks to the end of this line please?

Comment (mheon) [reply docs/tutorials/README.md]: Done

**Comment by TomSweeneyRedHat** [comment]
LGTM and thanks for touching up the prior line.  I spotted that this weekend and was going to touch that up.

**Comment by rhatdan** [comment]
/lgtm
/hold
cancel when tests pass.

**Comment by rhatdan** [comment]
Manually merging since this is not effecting tests.

**Comment by mheon** [comment]
Rebased

**Pull request merged**